packages/dds/counter/src/test/counter.spec.ts

```diff
@@ -65,6 +72,15 @@ describe("SharedCounter", () => {
                 assert.ok(fired1, "The event for first increment was not fired");
                 assert.ok(fired2, "The event for second increment was not fired");
             });
+
+            it("Applies a stashed op", () => {
```

Comment:
I'm a big fan of the mocha convention to write test suites and names to nest such that they read as sentences that give you an idea of what the test is for. (Outermost test suites being a scope thing is fine, imo).

So here I'd probably want an outer describe block that's just "applyStashedOp" and the test could be "immediately applies the op's increment locally"